modules/ROOT/pages/kafka/kafka-connector-studio.adoc

```diff
@@ -0,0 +1,113 @@
+= Kafka Studio Configuration - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+To configure a connector in Anypoint Studio:
+
+. Install the connector.
+. Configure the connector.
+. Configure an input source for the connector.
+
+== Install the Connector
+
+From within Studio:
+
+. In Studio, create a Mule Project.
+. In the Mule Palette, click *(X) Search in Exchange*.
+. In *Add Modules to Project*, type the name of the connector in the search field.
+. Click the connector name in *Available modules*.
+. Click *Add >*.
+. Click *Finish*.
+
+Or using Exchange:
```

Comment:
Did Kim OK the formatting of this? I would make these two diff procedures H2

== Install the Connector in Studio
== Install the Connector Using Exchange

Or similar